resources/wondersB.json

```diff
@@ -0,0 +1,394 @@
+[
+    {
+        "name": "RhodosB",
+        "id": 7,
+        "power": [
+            {
+                "effect": "produce",
+                "resources": [ ["o", 1] ],
+                "target": [],
+                "direction": ["self"]
+            }
+        ],
+        "stages": [
+            {
+                "cost": ["s", "s", "s", "s"],
```

Comment:
should be 3 stone

Reply:
nice catch, ty
addressed in 'fix rhodos b stage 1 resource cost` commit